ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -44,9 +45,79 @@ export default class StatusDropdown extends React.Component {
       showNotesModal: false,
       showTagsModal: false,
       selectedValue: this.props.issueTrackers[0].text,
+      jobList: [],
     };
   }
 
+  containsVismet = (title) => {
+    const contains = visualMetrics.find((metric) => title.includes(metric));
+    return contains;
+  };
+
+  anyAlertWithVideoResults = (alertSummary) => {
+    // For the moment the browsertime vismet are separate from the pageload ones
+    // that's why we need to filter them out. Also, for the moment we're retrieving
+    // the video results for regressions only
+    const any = alertSummary.alerts.filter(
+      (alert) =>
+        !this.containsVismet(alert.title) && alert.is_regression === true,
+    );
+    return any.length > 0;
+  };
+
+  isRelevantAlert = (alert) => {
+    return (
+      (alert.status !== summaryStatusMap.reassigned &&
+        alert.status !== summaryStatusMap.downstream &&
+        alert.status !== summaryStatusMap.invalid &&
+        alert.is_regression === true &&
+        alert.starred === true &&
+        !this.containsVismet(alert.title)) ||
+      (alert.status !== summaryStatusMap.reassigned &&
+        alert.status !== summaryStatusMap.downstream &&
+        alert.status !== summaryStatusMap.invalid &&
+        alert.is_regression === true &&
+        !this.containsVismet(alert.title))
+    );
+  };
+
+  updateSummary = (alertSummary, jobList) => {
```

Comment:
`updateSummary()` suggests we're doing a total revamp on the `alertSummary`. Which isn't right.
We're only enriching it with some task ids at some of the alerts.

Could we rename it so it's more accurate?